osf/models/base.py

```diff
@@ -108,6 +93,10 @@ class BaseModel(models.Model):
     class Meta:
         abstract = True
 
+    def to_storage(self):
```

Comment:
Can this be removed? It looks like we only used `to_storage()` in the old `TrashedFileNode.restore`, but the new version doesn't call it.